locale/en_US/locale.xml

```diff
@@ -20,26 +20,33 @@
 	<message key="plugins.generic.shibboleth.manager.settings.description"><![CDATA[<p>With this plugin enabled, a Shibboleth single sign-on service can be used to register and authenticate users. You must have set up and configured the local service provider (SP).</p>]]></message>
 
 	<message key="plugins.generic.shibboleth.manager.settings.shibbolethWayfUrl">Shibboleth SP path</message>
-	<message key="plugins.generic.shibboleth.manager.settings.shibbolethWayfUrlRequired">Please enter the path of the Shibboleth wayfinder service on this server.</message>
+        <message key="plugins.generic.shibboleth.manager.settings.shibbolethWayfUrlRequired">Please enter the path of the Shibboleth wayfinder service on this server. (e.g. Shibboleth.sso/Login)</message>
 
-	<message key="plugins.generic.shibboleth.manager.settings.shibbolethHeaderUin">Header with Shibboleth UIN</message>
+	<message key="plugins.generic.shibboleth.manager.settings.shibbolethHeaderUin">Server Environment Variable ($_SERVER) with Shibboleth UIN</message>
```

Comment:
The new instruction seems a bit confusing to me, but obviously the old instruction seemed confusing to you.  What we want to know here (and elsewhere) is what HTTP Header contains the data value, specifically in how that HTTP Header is expressed from a system environment variable as represented in the $_SERVER superglobal.  Perhaps:
> HTTP Header ($_SERVER environment variable key) ....

Reply:
I like that compromise.